storage/rocksdb/rdb_datadic.cc

```diff
@@ -1416,14 +1416,19 @@ int Rdb_key_def::unpack_record(TABLE *const table, uchar *const buf,
   // For secondary keys, we expect the value field to contain unpack data and
```

Comment:
We should probably update the comment here to say, we expect for SKs to have index flags in the beginning as well.(which is just TTL only at the moment)

Reply:
Ack,makes sense.  I'll update and re-push later.